data/netflix.com-view.js

```diff
@@ -0,0 +1,10 @@
+
+/**
+ * MediaKeys namespace.
+ */
+if (typeof MediaKeys == "undefined") var MediaKeys = {};
+//
+MediaKeys.playButton = "//div[@class='player-control-button player-play-pause play icon-player-play']";
+MediaKeys.pauseButton = "//div[@class='player-control-button player-play-pause pause icon-player-pause']";
+MediaKeys.skipButton = "//div[@class='player-control-button player-next-episode container-icon-player-next-episode']";
```

Comment:
I strongly encourage using the xpath 'contains' method for matching class attributes like you did below. Play could be as simple as //div[contains(@class,' play ')]